webapp/components/admin_console/cluster_settings.jsx

```diff
@@ -0,0 +1,187 @@
+// See License.txt for license information.
+
+import React from 'react';
+
+import AdminSettings from './admin_settings.jsx';
+import BooleanSetting from './boolean_setting.jsx';
+import TextSetting from './text_setting.jsx';
+
+import {FormattedMessage, FormattedHTMLMessage} from 'react-intl';
+import SettingsGroup from './settings_group.jsx';
+import ClusterTableContainer from './cluster_table_container.jsx';
+
+import AdminStore from 'stores/admin_store.jsx';
+import * as Utils from 'utils/utils.jsx';
+
+export default class ClusterSettings extends AdminSettings {
+    constructor(props) {
+        super(props);
+
+        this.getConfigFromState = this.getConfigFromState.bind(this);
+        this.renderSettings = this.renderSettings.bind(this);
+    }
+
+    getConfigFromState(config) {
+        config.ClusterSettings.Enable = this.state.enable;
+        config.ClusterSettings.InterNodeListenAddress = this.state.interNodeListenAddress;
+
+        config.ClusterSettings.InterNodeUrls = this.state.interNodeUrls.split(',');
+        config.ClusterSettings.InterNodeUrls = config.ClusterSettings.InterNodeUrls.map((url) => {
+            return url.trim();
+        });
+
+        if (config.ClusterSettings.InterNodeUrls.length === 1 && config.ClusterSettings.InterNodeUrls[0] === '') {
+            config.ClusterSettings.InterNodeUrls = [];
+        }
+
+        return config;
+    }
+
+    getStateFromConfig(config) {
+        const settings = config.ClusterSettings;
+
+        return {
+            enable: settings.Enable,
+            interNodeUrls: settings.InterNodeUrls.join(', '),
+            interNodeListenAddress: settings.InterNodeListenAddress,
+            showWarning: false
+        };
+    }
+
+    renderTitle() {
+        return (
+            <h3>
+                <FormattedMessage
+                    id='admin.advance.cluster'
+                    defaultMessage='High Availability'
+                />
+            </h3>
+        );
+    }
+
+    overrideHandleChange = (id, value) => {
+        this.setState({
+            showWarning: true
+        });
+
+        this.handleChange(id, value);
+    }
+
+    renderSettings() {
+        const licenseEnabled = global.window.mm_license.IsLicensed === 'true' && global.window.mm_license.Cluster === 'true';
+        if (!licenseEnabled) {
+            return null;
+        }
+
+        var configLoadedFromCluster = null;
+
+        if (AdminStore.getClusterId()) {
+            configLoadedFromCluster = (
+                <div
+                    style={{marginBottom: '10px'}}
+                    className='alert alert-warning'
+                >
+                    <i className='fa fa-warning'></i>
+                    <FormattedMessage
+                        id='admin.cluster.loadedFrom'
+                        defaultMessage='This config file was loaded from Node ID "{clusterId}".  If you are accessing the System Console through a load balancer you may experience issues.'
```

Comment:
@coreyhulen wondering if we could discuss this message to say a little more about the situation?